packages/template-retail-react-app/app/components/product-tile/index.test.js

```diff
@@ -51,42 +62,74 @@ test('Remove from wishlist cannot be muti-clicked', () => {
     expect(onClick).toHaveBeenCalledTimes(1)
 })
 
-test('Renders variation selection swatches', () => {
+test('Renders variant details based on the selected swatch', async () => {
+    useBreakpointValue.mockReturnValue(true)
```

Comment:
Can we add cases where this value is false for mobile behavior?

Reply:
I feel those should be part of the swatch. I will add them there

Reply:
The swatch tests are all broken. I will create a separate ticket for that. For now, I have added a test for mobile behavior